src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        Truck truck1 = new Truck("tanker", 112233, 20);
+        Truck truck2 = new Truck("tanker", 445566, 25);
+        Truck truck3 = new Truck("tanker", 778899, 50);
+        return List.of(truck1, truck2, truck3);
```

Comment:
```suggestion
        return List.of(new Truck("tanker", 778899, 50),
```
You can simplify the code by directly creating new instances of the Truck class within the List.of method call, without the need for storing them in temporary variables

Reply:
in this task totally agree. But in future, if I use these variables somewhere, it'll better to store them?

Reply:
You may introduce variables if they are reused or if they add clarity to code. So for sure there are possible situations where creation of variable will provide benefits.
But if not then it's better to use less variables :)